mcp/1.1/modules/ROOT/pages/mcp-connector-reference.adoc

```diff
@@ -0,0 +1,1120 @@
+| Reconnection Delay Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time Unit for the reconnectionDelay |  MILLISECONDS | 
+| Tls Context a| <<Tls>> |  |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+
+[[Client_StreamableHttpClient]]
+===== Streamable HTTP Client
+
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Request Timeout a| Number |  Sets the duration to wait for server responses before timing out requests. |  30 | 
+| Request Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time Unit for the requestTimeout parameter |  SECONDS | 
+| Authentication a| HttpRequestAuthentication |  |  | 
+| Server Url a| String |  The base URL for the target MCP server |  | *Yes*
+| Mcp Endpoint Path a| String |  |  /mcp | 
+| Tls Context a| <<Tls>> |  |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+|===
+
+== Supported Operations
+
+* <<CallTool>> 
+* <<ListResources>> 
+* <<ListTools>> 
+* <<Ping>> 
+* <<ReadResource>> 
+
+[[Server]]
+=== Server
+
+Configuration element to act as an MCP server.
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | Name for this configuration. Connectors reference the configuration with this name. | | *Yes*
+| Connection a| * <<Server_SseServer, Sse Server>> 
+* <<Server_StreamableHttpServer, Streamable Http Server>> 
+ | The connection types to provide to this configuration. | | *Yes*
+| Server Name a| String |  Name this server uses to identify itself during the protocol initialization phase. This isn't the same as the name given to this config element (which is later referenced through config-ref parameters). This name doesn't need to be unique.  |  | *Yes*
+| Server Version a| String | Version this server uses to identify itself during the protocol initialization phase. This doesn't need to match the connector or Mule application version.  |  | *Yes*
+| Name a| String |  ID used to reference this configuration. |  | *Yes*
+|===
+
+
+
+==== Connection Types
+
+[[Server_SseServer]]
+===== SSE Server
+
+
+Exposes the MCP server using the SSE transport, as defined in https://spec.modelcontextprotocol.io/specification/2024-11-05/basic/transports/#http-with-sse[] IMPORTANT: This connection type only works if Mule is started with the mule.http.service.implementation=NETTY system property.
```

Comment:
can this note be removed? bc NETTY is supposed to be removed.

Reply:
YES